tooling/cli.rs/src/info.rs

```diff
@@ -576,3 +631,134 @@ impl Info {
     Ok(())
   }
 }
+
+fn get_package_manager(app_dir: &VfsPath) -> crate::Result<PackageManager> {
+  let use_yarn = app_dir.join("yarn.lock")?.exists().unwrap_or_default();
+
+  let use_npm = app_dir
+    .join("package-lock.json")?
+    .exists()
+    .unwrap_or_default();
+
+  let use_pnpm = app_dir.join("pnpm-lock.yaml")?.exists().unwrap_or_default();
+
+  if !use_npm && !use_pnpm && !use_yarn {
+    return Err(anyhow::anyhow!(
+      "no lock file found, please first run your package manager install"
+    ));
+  }
+
+  if use_npm && (use_pnpm || use_yarn)
+    || use_pnpm && (use_npm || use_yarn)
+    || use_yarn && (use_npm || use_pnpm)
+  {
+    return Err(anyhow::anyhow!(
+      "only one package mangager should be used, but found npm {}, pnpm {}, yarn {}\nplease remove unused package manager lock files",
```

Comment:
can this also be changed to only mention found package managers, instead of  `found npm true, yarn false, pnpm true` it should be `found npm and pnpm`

Reply:
Fixed